src/hooks/useRemainingRelays.ts

```diff
@@ -20,7 +20,11 @@ const useRemainingRelays = () => {
   const chainId = useChainId()
   const safeAddress = useSafeAddress()
 
-  return useAsync(() => fetchRemainingRelays(chainId, safeAddress), [chainId, safeAddress])
+  return useAsync(() => {
+    if (!safeAddress) return
```

Comment:
Seeing as we caught the bug by accident, let's be safe and check for the presence of `chainId` as well. Can you add a test for it too please?

```suggestion
    if (!chainId || !safeAddress) return
```

Reply:
I initially had this logic. but then I verified we never return a falsy chainId